components/support/nimbus-fml/src/client/inspector.rs

```diff
@@ -66,6 +66,12 @@ impl FmlFeatureInspector {
     pub fn get_errors(&self, string: String) -> Option<Vec<FmlEditorError>> {
         self.get_first_error(string).map(|e| vec![e])
     }
+
+    pub fn get_structure_hash(&self) -> String {
```

Comment:
I definitely agree that "structure" isn't as clear as I'd like it to be

Reply:
"Schema" does seem like a good alternative, since we use "definition" in other places